### DISCUSSION TITLE
Add support for JPG drag and drop

### DISCUSSION THREAD

**Discussion by QuietNoise** [Comfy-Org/ComfyUI#1885] [Ideas]
In same way you allow to drag and drop PNG files and import Generation Data from them please allow importing same data from JPG files too.

**Comment by phineas-pta** [comment]
technically u can

but online jpg are usually without any metadata so this feature isnt useful

**Reply by QuietNoise** [reply]
Usually yes, but some platforms that are targetting AI creators retain this metadata in jpeg files just for the purpose of easily recreating the artwork or its likeness.  I.e. https://civitai.com/ is amongst the most popular ones that serves jpgs with generation data in them.
Automatic1111 webgui has support for jpgs from the start for this purpose too and it's tremendously helpful when one can drag the jpg image to investigate its parameters.
